user/advanced-topics/managing-vm-kernels.md

```diff
@@ -287,7 +293,11 @@ Install distribution kernel image, kernel headers and the grub.
 sudo apt install linux-image-amd64 linux-headers-amd64 grub2 qubes-kernel-vm-support
 ~~~
 
-If you are doing that on a qube based on "Debian Minimal" template, a grub gui will popup during the installation, asking you where you want to install the grub loader. You must select /dev/xvda (check the box using the space bar, and validate your choice with "Enter".)
+If you are doing that on a qube based on "Debian Minimal" template, a grub gui will popup during the installation, asking you where you want to install the grub loader. You must select /dev/xvda (check the box using the space bar, and validate your choice with "Enter".) If this popup does not appear during the installation, you must manually setup `grub2` by running:
+
+~~~
+sudo grub-install /dev/xvda
+~~~
```

Comment:
If I'm reading this correctly, this repeats the command that you instructed the user to run above. Won't that be confusing? Also, should we really be telling the user the run this command above, at the start, when we later mention the GUI option?